source/tests/yats-test/pipeline_test.cpp

```diff
@@ -94,11 +94,11 @@ TEST(pipeline_test, use_external_input)
 
     yats::scheduler scheduler(std::move(pipeline));
 
-    writer(15);
+    writer(15, true);
```

Comment:
could we maybe have this as a default parameter? I can only see usage with true, so let's have true as the default

Reply:
I would default to false or otherwise add a function with a different name like write_last